charts/camunda-platform-8.6/templates/camunda/secret-identity.yaml

```diff
@@ -0,0 +1,12 @@
+{{- if and (.Values.global.identity.auth.enabled) (typeIs "string" .Values.global.identity.auth.identity.existingSecret) }}
```

Comment:
There is no need for this file as we set the secret directly in the app config, and it's used in only one place: Identity (unlike the other apps, where the same secret is used in Identity and the corresponding component, like Operate).

Reply:
has been removed